subprojects/docs/docs.gradle

```diff
@@ -147,6 +147,23 @@ samples {
             target = "utilities"
         }
 
+        javaModuleInfoForListLibrary {
+            target = "list"
+        }
+        javaModuleInfoForUtilitiesLibrary {
+            target = "utilities"
+        }
+        javaModuleInfoForApplication {
+            target = "application"
+        }
+        javaJunit5ModuleInfoForUtilitiesLibrary {
+            target = "utilities"
+        }
+        javaJunit5ModuleInfoForApplication {
+            target = "application"
+        }
```

Comment:
I see that this best matches the structure but it just hurts to see that we need more boilerplate code for those files than the files have content. Do you think we should merge them into one template?

Reply:
I see your point. I think I like to keep following the structure for now and revisit this when we added more JVM samples. I think at one point we should at least organize the `templates` folder with subfolders. And maybe there is also a way to improve this 'sample definition' DSL so that we can keep the structure, but have less boilerplate to write. (E.g. If we have a common structure, we can maybe just read the folder without having to explicitly define templates)